gwcs/wcs.py

```diff
@@ -1678,16 +1686,14 @@ def _calc_approx_inv(self, max_inv_pix_error=5, inv_degree=None, npoints=16):
             # A bounding_box is needed to proceed.
             return
 
-        (xmin, xmax), (ymin, ymax) = self.bounding_box
-        crpix1 = (xmax - xmin) // 2
-        crpix2 = (ymax - ymin) // 2
-        crval1, crval2 = self.forward_transform(crpix1, crpix2)
+        crpix = np.mean(self.bounding_box, axis=1, dtype=np.int)
```

Comment:
Why not do this computation the same way between lines 1439,40 and here?

Reply:
Why do it? In fact, I can probably remove `int` from here. I do not need separate `crpix1,2` here and I do not need to round anything since this is not going to go into a FITS header.